linux_os/guide/system/network/network-wireless/wireless_software/wireless_disable_interfaces/oval/ubuntu.xml

```diff
@@ -0,0 +1,16 @@
+<def-group>
+  <definition class="compliance" id="wireless_disable_interfaces" version="1">
+    {{{ oval_metadata("All wireless interfaces should be disabled.") }}}
+    <criteria>
+      <criterion comment="query /proc/net/wireless" test_ref="test_wireless_disable_interfaces" />
+    </criteria>
+  </definition>
+  <ind:textfilecontent54_test check="all" check_existence="none_exist" comment="query /proc/net/wireless" id="test_wireless_disable_interfaces" version="1">
+    <ind:object object_ref="object_wireless_disable_interfaces" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="object_wireless_disable_interfaces" version="1">
```

Comment:
this check is only identifying if there's any wireless interface, it is not checking if they are disabled.
The shared oval is better

Reply:
I did this test to check if the interface is still listed:
```
# cat /proc/net/wireless 
Inter-| sta-|   Quality        |   Discarded packets               | Missed | WE
 face | tus | link level noise |  nwid  crypt   frag  retry   misc | beacon | 22
wlp2s0: 0000   45.  -65.  -256        0      0      0      0      1        0

# ip link set wlp2s0 down

# cat /proc/net/wireless 
Inter-| sta-|   Quality        |   Discarded packets               | Missed | WE
 face | tus | link level noise |  nwid  crypt   frag  retry   misc | beacon | 22
```

I did find an issue with the regex though.